tests/orm/test_querybuilder.py

```diff
@@ -629,6 +629,32 @@ def test_direction_keyword(self):
         res2 = {item[1] for item in qb.all()}
         self.assertEqual(res2, {d2.id, d4.id})
 
+    @staticmethod
+    def test_flat():
+        """Test the `flat` keyword for the `.iterall()`, `.all()` and `.one()` methods."""
```

Comment:
Mmm how is this testing `.iterall()` and `.one()`? Is this some copy/paste leftover?

Reply:
Yeah, this is just a mistake that I left in. I originally thought I was going to implement it for all, but for `one` it would break backwards-compatibility and for `iterall` it does not make sense